source/ada/lsp-ada_handlers.adb

```diff
@@ -2114,26 +2115,147 @@ package body LSP.Ada_Handlers is
          is
             use Libadalang.Common;
 
-            Token : Token_Reference := First_Token (Node.Unit);
-            Name  : constant Wide_Wide_String :=
+            Token   : Token_Reference := First_Token (Node.Unit);
+            Name    : constant Wide_Wide_String :=
               Ada.Strings.Wide_Wide_Unbounded.To_Wide_Wide_String
                 (Get_Last_Name (Name_Node));
-            Span  : LSP.Messages.Span;
+            C       : LSP.Messages.TextEdit;
+            Span    : LSP.Messages.Span;
+            Current : Token_Reference;
+            Diff    : Integer;
+
+            Box_Line : constant Wide_String (1 .. 80) := (others => '-');
+
+            procedure Include_Result (Edit : LSP.Messages.TextEdit);
+            --  Add prepared TextEdit to Result;
+
+            function Process_Box return Boolean;
+            --  Check whether Current is box header/fother and modify it.
+            --  Return False when the searching cycle should be stopped.
+
+            --------------------
+            -- Include_Result --
+            --------------------
+
+            procedure Include_Result (Edit : LSP.Messages.TextEdit) is
+            begin
+               if not Response.result.changes (Uri).Contains (Edit) then
+                  Response.result.changes (Uri).Append (Edit);
+               end if;
+            end Include_Result;
+
+            -----------------
+            -- Process_Box --
+            -----------------
+
+            function Process_Box return Boolean is
+               use Langkit_Support.Text;
+               use Langkit_Support.Slocs;
+
+            begin
+               if Current = No_Token then
+                  return False;
+               end if;
+
+               case Kind (Data (Current)) is
+                  when Ada_Whitespace =>
+                     return True;
+
+                  when Ada_Comment =>
+                     declare
+                        T : constant Text_Type := Text (Current);
```

Comment:
`T` is too short

Reply:
Agreed

Reply:
@AnthonyLeonardoGracio This is already fixed, that why you can see
source/ada/lsp-ada_handlers.adb Outdated

the keyword is Outdated